README.md

```diff
@@ -147,6 +147,17 @@ new Vue({
 })
 ```
 
+```js
+// Nuxt 3
+export default defineNuxtConfig({
+  // ... other options
+  modules: [
+    // ...
+    '@pinia/nuxt',
+  ],
+})
+```
```

Comment:
For more detailed instructions, including [Nuxt configuration](https://pinia.vuejs.org/ssr/nuxt.html#nuxt-js), check the [Documentation](https://pinia.vuejs.org).